packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -70,6 +70,7 @@ const tabs: VerticalTabItemProps[] = [
       //
       { name: "webhooks", href: "/settings/developer/webhooks" },
       { name: "api_keys", href: "/settings/developer/api-keys" },
+      { name: "self_hosting", href: "/settings/developer/self-hosting" },
```

Comment:
this has to be filtered by isCalcom === true

we only wanna show this navigation item on app.cal.com